tracpro/baseline/tests/test_models.py

```diff
@@ -86,26 +86,13 @@ def setUp(self):
 
     def test_baseline_all_regions(self):
         """ Answers were 10, 20 and 30: Total should be 10 + 20 + 30 = 60 """
-        baseline_dict, dates = self.baselineterm.get_baseline(regions=None, region_selected=0)
-
-        self.assertEqual(len(baseline_dict), 2)  # Two regions, two sets of baseline values
-        # Two answers, 10 + 20 = 30
-        self.assertEqual(
-            baseline_dict[self.region1.name]["values"],
-            [30])
-        # One answer, 30 = 30
-        self.assertEqual(
-            baseline_dict[self.region2.name]["values"],
-            [30])
+        baseline_total, dates = self.baselineterm.get_baseline(regions=None, region_selected=0)
```

Comment:
Alternatively adding `None` as the default for `region_selected` on the method itself.